portal/gil/templates/gil/base.html

```diff
@@ -44,6 +44,23 @@
         {{_("Loading")}}... &nbsp;
   </div>
   <section id="mainHolder">
+    <input type="hidden" id="currentUserId" value="{{user.id if user}}" />
+    {% if user and (user.has_role(ROLE.PATIENT) or user.has_role(ROLE.PARTNER)) %}
+      <input type="hidden" id="interventionMenu" value="true" />
+    {% endif %}
+    {% if timed_out %}
+      <input type="hidden" id="sessionTimedOut" value="true" />
+    {% endif %}
+    {% if request.args.get("disableLinks", "") %}
```

Comment:
Does this querystring parameter control more than this one HTML element?
Would you be able to check for this querystring parameter with JS?